recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc.sh

```diff
@@ -343,8 +343,10 @@ tdx_enc_clear_keys_keyring() {
 
 # umount partition
 tdx_enc_partition_umount() {
-    tdx_enc_log "Unmounting dm-crypt partition..."
-    umount "${TDX_ENC_STORAGE_MOUNTPOINT}"
+    for mnt in $(lsblk /dev/mapper/"${TDX_ENC_DM_DEVICE}" -n -o MOUNTPOINTS); do
```

Comment:
Change here looks good. Just a small typo in the commit message: `paritition` -> `partition`.

Reply:
Nice catch!